src/managers/epicenter-channel-manager.js

```diff
@@ -66,9 +66,9 @@ var EpicenterChannelManager = classFrom(ChannelManager, {
     },
 
     /**
-     * Create and return a publish/subscribe channel (from the underlying [Channel Manager](../channel-manager/)) for the given [group](../../../glossary/#groups). The group must be one of the groups in this account (team) and project.
+     * Create and return a publish/subscribe channel (from the underlying [Channel Manager](../channel-manager/)) for the given [group](../../../glossary/#groups). The group must be one of the groups in this team and project.
```

Comment:
The last sentence sounds a bit redundant. "The group must exist in the account and project provided"?